src/jltypes.c

```diff
@@ -2528,19 +2527,8 @@ static jl_tvar_t *tvar(const char *name)
                           (jl_value_t*)jl_any_type);
 }
 
-void export_small_typeof(void)
-{
-    void *copy;
-#ifdef _OS_WINDOWS_
-    jl_dlsym(jl_libjulia_handle, "small_typeof", &copy, 1);
-#else
-    jl_dlsym(jl_libjulia_internal_handle, "small_typeof", &copy, 1);
-#endif
-    memcpy(copy, &small_typeof, sizeof(small_typeof));
-}
-
 #define XX(name) \
-    small_typeof[(jl_##name##_tag << 4) / sizeof(*small_typeof)] = jl_##name##_type; \
```

Comment:
Please revert these changes.

Reply:
The rename? Do you want to keep it named `small_typeof` across the board?

Reply:
Okay sure thing, but can you explain to me how `export_small_typeof` is supposed to work?

From what I could see on Linux `small_typeof` was picking up the libjulia copy everywhere, and this exported copy in libjulia-internal seemed to be unused

Reply:
This isn't intended to be exported. Though some platforms might have needed JL_HIDDEN to avoid the ODR issue and pick up the correct copy of it.